osc-staging.py

```diff
@@ -109,28 +109,40 @@ def do_staging(self, subcmd, opts, *args):
     ${cmd_option_list}
 
     "accept" will accept all requests in
-        openSUSE:Factory:Staging:<LETTER> (into Factory)
+        $PROJECT:Staging:<LETTER> into $PROJECT
 
-    "acheck" will check if it's safe to accept new staging projects
-        As openSUSE:Factory is syncing the right package versions between
-        /standard, /totest and /snapshot, it's important that the projects
+    "acheck" will check if it is safe to accept new staging projects
+        As $PROJECT is syncing the right package versions between
+        /standard, /totest and /snapshot, it is important that the projects
         are clean prior to a checkin round.
 
+    "adi" will list already staged requests, stage new requests, and supersede
+        requests where applicable. New adi stagings will be created for new
+        packages based on the grouping options used. The default grouping is by
+        source project. When adi stagings are ready the request will be marked
+        ready, unstaged, and the adi staging deleted. If the user has permission
+        to accept, the requests marked ready will be accepted.
```

Comment:
the adi command does not accept any requests... it only sets all 'reviews' as accepted; once 'adi' is completed, the submit request is in state 'new'

osc staging accept (with or without letter) accepts the requests finally